test/TesterAzureUtils/TesterAzureUtils.csproj

```diff
@@ -0,0 +1,160 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Tester is a weird name and we should never have chosen it. This project should be called "Tests.AzureUtils" or, using the normal convention, "OrleansAzureUtils.Tests". I'm fine with leaving this as-is, we can rename test projects later if we want.